components/admin_console/billing/billing_subscriptions/billing_subscriptions.tsx

```diff
@@ -1,24 +1,43 @@
 // Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
 // See LICENSE.txt for license information.
 
-import React, {useState} from 'react';
-import {FormattedMessage} from 'react-intl';
+import React, {useState, useEffect} from 'react';
+import {FormattedMessage, useIntl} from 'react-intl';
 import {Tooltip} from 'react-bootstrap';
 
+import {PreferenceType} from 'mattermost-redux/types/preferences';
+
```

Comment:
Nit: extra spacing. We can group all the `mattermost-redux` imports together

Reply:
I'm not sure why VSCode always does this to me. Fixed.